src/main/java/com/mealmentor/enterprise/MealMentorController.java

```diff
@@ -71,18 +76,18 @@ public ResponseEntity deleteMealItem(@PathVariable("id") String id) {
 
     }
 
-    @RequestMapping(value = "/",method = RequestMethod.GET)
-    public String read(Model model){
+    @RequestMapping(value = "/", method = RequestMethod.GET)
+    public String read(Model model) {
         model.addAttribute("recipe", new Recipe());
         return "start";
     }
 
 
     @GetMapping(value = "/searchRecipe")
     @ResponseBody
-    public String searchReceipe(@RequestParam(value="searchTerm", required=false, defaultValue="None")  String searchTerm, Model model) throws IOException {
+    public String searchRecipe(@RequestParam(value = "searchTerm", required = false, defaultValue = "None") String searchTerm, Model model) throws IOException {
```

Comment:
Changed the method name to make it the same as getting request call name `searchRecipe` for consistency and keeping the code clean.

Clean coding naming conventions: https://www.baeldung.com/java-clean-code

Reply:
Ahhh, spelling error.  Good catch.